app/mixins/Analytics.js

```diff
@@ -0,0 +1,11 @@
+var analytics = require("webmaker-analytics");
+
+// This mixin adds analytics event track capabilities to a component
+module.exports = {
+
+  event: function(action, options) {
```

Comment:
Agreed. I think `this.event(...)` is pretty confusing. I'm leaning towards just including as a plain JS lib.